README.md

```diff
@@ -1,29 +1,41 @@
 # Validations with `form_tag`
 
-## Objectives
+# Objectives
 
-- render or redirect based on validation of instance in create/update
+- Render or redirect based on validation of instance in create/update
 - Prefill in form values based on an instance
 - Print out full error messages based on an invalid instance
-- introspect on errors for a field
-- apply an error class to a field conditionally based on errors on a field
-
-## Notes
-
-Just give them the models and migrations.
-
-They have to add
-
-- Validations
-- Routes
-- Controllers
-- Actions
-- Form Views
-
-make sure they are redirectnig and rendering correctly
-make sure they are calling save and update correctly
-make sure they print out full messages for an error summary on invalid forms
-fields should be prefilled
-error class on inputs
+- Introspect on errors for a field
+- Apply an error class to a field conditionally based on errors on a field
+
+# Requirements
+
+## Validations
+
+1. Add validations to `Author` such that...
+  1. `name` is not blank
+  1. `email` is unique
+  1. `phone_number` is exactly 10 digits long
+1. Add validations to `Post` such that...
+  1. `title` is not blank
+  1. `content` is at least 100 characters long
+  1. `category` is either `"Fiction"` or `"Non-Fiction"`
+
+## Basic Routes & Controllers
+
+1. Create controllers for both models.
+1. Create `show`, `new`, `edit`, `create`, and `update` routes for both models.
+1. Define controller actions for `show`, `new`, and `edit`.
```

Comment:
Hey @mcantor, so we already had students build the stuff like the the simple routes, actions and such in the last lab they did. So I would include the solution to the last lab they did in here as the starter code. Then the point of this lab is to just have them render those errors to students.